environments/sbox.tfvars

```diff
@@ -6,9 +6,9 @@ enable_ssl                 = true
 ssl_mode                   = "AzureKeyVault"
 certificate_key_vault_name = "cftapps-sbox"
 
-shutter_storage = "default"
-cdn_sku         = "Standard_Verizon"
-shutter_rg      = "shutter-app-sbox-rg"
+shutter_storage     = "default"
+cdn_sku             = "Standard_Verizon"
+resource_group_name = "shutter-app-sbox-rg"
```

Comment:
Why is the change in variable name?
This should be 
```
shutter_rg      = "shutter-app-sbox-rg"
```

Reply:
https://github.com/hmcts/azure-platform-terraform/pull/97/files/af53a8fd9c65bd1b888c62c1da5f24d91538a19e#r376359725